apps/api/src/app/bridge/usecases/sync/sync.usecase.ts

```diff
@@ -215,24 +215,17 @@ export class Sync {
         triggerIdentifier: workflow.workflowId,
         __source: WorkflowCreationSourceEnum.BRIDGE,
         steps: this.mapSteps(workflow.steps),
-        /** @deprecated */
-        inputs: {
-          schema: workflow.controls?.schema || workflow.inputs.schema,
-        },
         controls: {
-          schema: workflow.controls?.schema || workflow.inputs.schema,
+          schema: workflow.controls?.schema,
         },
         rawData: workflow as unknown as Record<string, unknown>,
-        payloadSchema:
-          (workflow.payload?.schema as Record<string, unknown>) ||
-          /** @deprecated */
-          (workflow.options?.payloadSchema as Record<string, unknown>),
+        payloadSchema: workflow.payload?.schema,
         active: isWorkflowActive,
         description: this.getWorkflowDescription(workflow),
-        data: this.castToAnyNotSupportedParam(workflow).options?.data,
+        data: this.castToAnyNotSupportedParam(workflow)?.data,
         tags: this.getWorkflowTags(workflow),
-        critical: this.castToAnyNotSupportedParam(workflow.options)?.critical ?? false,
-        preferenceSettings: this.castToAnyNotSupportedParam(workflow.options)?.preferenceSettings,
```

Comment:
i think the WorkflowDefine still has `option` param, any reason for this change?

Reply:
ok but DiscoverWorkflowOutput and WorkflowDefine are "different" objects, if what you say is right we need to:
1. remove options from WorkflowDefine.
2. remove all the mapping in createWorkflow and updateWorkflow.

Reply:
Good catch! I have removed those redundant properties from the `SyncCommand`